sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/TimeSeriesIdTests.cs

```diff
@@ -99,24 +100,10 @@ public void TimeSeriesId_GetIdForStringKeys()
             var tsiId = new TimeSeriesId("B17", "F1", "R400");
 
             // Act
-            var idAsString = tsiId.GetId();
+            var idAsString = tsiId.ToString();
 
             // Assert
-            idAsString.Should().Be($"Time Series Id key 1: B17\nTime Series Id key 2: F1\n" +
-                $"Time Series Id key 3: R400");
-        }
-
-        [Test]
-        public void TimeSeriesId_GetIdForObjectKeys()
-        {
-            // Arrange
-            var tsiId = new TimeSeriesId(true, 1);
-
-            // Act
-            var idAsString = tsiId.GetId();
-
-            // Assert
-            idAsString.Should().Be($"Time Series Id key 1: True\nTime Series Id key 2: 1");
+            idAsString.Should().Be("[\"B17\",\"F1\",\"R400\"]");
```

Comment:
consider breaking the 3 parameters above into variables, and then concat this string using those variables.

Reply:
Good idea